integration_test/gce/gce_testing.go

```diff
@@ -1550,8 +1550,8 @@ func InstallGsutilIfNeeded(ctx context.Context, logger *log.Logger, vm *VM) erro
 
 	// SUSE seems to be the only distro without gsutil, so what follows is all
 	// very SUSE-specific.
-	if !IsSUSE(vm.ImageSpec) {
-		return installErr("gsutil", vm.ImageSpec)
+	if vm.OS.ID != "opensuse" && vm.OS.ID != "sles" && vm.OS.ID != "sles_sap" {
```

Comment:
Why are we not changing `IsSUSE` to use `vm.OS.ID` and then calling that?

Reply:
This was the approach we ended up using in https://github.com/GoogleCloudPlatform/ops-agent/pull/1720 .